src/LayerStyler/utils.js

```diff
@@ -0,0 +1,25 @@
+import debounce from 'lodash.debounce'
+import ugh from 'ugh'
+
+/**
+ * Bind multiple move listeners with the same callback
+ * @function
+ * @param {ol.Map} map - The openlayers map to which the events are bound
+ * @param {Function} callback - The callback invoked when a `change:size`, `change:resolution` or a `change:center` event was fired
+ * @param {Object} [thisObj] - The object to use as `this` in the event listeners.
+ * @returns {ol.EventsKey[]} Array of openlayers event keys for unsetting listener events (See: https://openlayers.org/en/v4.6.5/apidoc/ol.Observable.html#.unByKey)
+ */
+export const addMovementListener = (map, callback, thisObj) => {
+  if (typeof callback !== 'function') return ugh.error('addMovementListener requires a valid openlayers map & callback function')
+
+  // If performance becomes an issue with catalog layers & far zoom level, these debounce levels can be adjusted
+  const slowDebounce = debounce(callback, 0)
+  const fastDebounce = debounce(callback, 0)
+  const keys = [
+    map.on('change:size', slowDebounce, thisObj),
+    map.getView().on('change:resolution', slowDebounce, thisObj),
+    map.getView().on('change:center', fastDebounce, thisObj)
+  ]
+
+  return keys
+}
```

Comment:
this whole util exists in the popup utils & this export will collide with that one....finally a case to hoist a common util??